src/app/file-browser/components/file-viewer/file-viewer.component.html

```diff
@@ -38,7 +38,15 @@
         ></pr-thumbnail>
         <pr-video [item]="currentRecord" *ngIf="isVideo"></pr-video>
         <pr-audio [item]="currentRecord" *ngIf="isAudio"></pr-audio>
-        <iframe [src]="documentUrl" *ngIf="isDocument && documentUrl"></iframe>
+        <iframe
+          [title]="
+            currentRecord.altText
+              ? currentRecord.altText
+              : currentRecord.displayName
```

Comment:
This pattern is used multiple times throughout this PR. Maybe it would make sense to extract this out into a Pipe? I'm not sure if it's fully necessary, but I think it'd probably read easier, be testable, and allow us to change the altText algorithm later in one place. Then we could just write:
```
[title]="currentRecord | getAltText"
```
Or something similar.

Reply:
thanks! that is a good idea!